sonic-slave-stretch/apt-retries-count

```diff
@@ -0,0 +1 @@
+Acquire::Retries "20";
```

Comment:
Could you reuse this file, or provide a centralized place to config 20. Assume someone will change the number in future.

Reply:
Reused.

Please approve and merge